src/java/org/apache/cassandra/index/SecondaryIndexManager.java

```diff
@@ -1706,6 +1754,7 @@ public void handleNotification(INotification notification, Object sender)
                                      indexes.values()
                                             .stream()
                                             .filter(Index::shouldBuildBlocking)
+                                            .filter(i -> !i.isSSTableAttached())
```

Comment:
If we're here, it's either after streaming or via `SSTableImporter`. SAI will already have written its indexes in the streaming case (entire file streaming, or writing incrementally via `SSTableFlushObserver`), and this patch redirects the importer case.

So, this is here to just make sure we ignore SAI indexes.

Reply:
This will also ignore SASI, won't it?

Reply:
Yeah...I think for the time being, I'm going to just have `SASIIndex` return false from `isSSTableAttached()` (the default). There's no point in changing it and worrying about whether SASI works the same way for `SSTableImporter` w/ more tests, etc.

By the way, `SSTableFlushObserver` also ensures that SAI indexes build when they should for the SSTable scrubber, upgrader, and splitter tools.